package.json

```diff
@@ -26,8 +26,8 @@
     "format:check": "prettier --config ./.prettierrc \"projects/{step-core/src/lib,step-frontend/src/lib,step-app/src}/!(angularjs)/**/*{.ts,.html,.scss}\"",
     "prepare": "husky install",
     "push": "npm publish projects/step-core/ && npm publish dist/step-frontend/*.tgz",
-    "openapi:generate:nhy": "./generate-openapi-client.sh https://stepos-nhy.exense.ch/rest/private-openapi.json",
-    "openapi:generate:master": "./generate-openapi-client.sh https://stepos-master.exense.ch/rest/private-openapi.json",
+    "openapi:generate:nhy": "./generate-openapi-client.sh https://stepos-nhy.stepcloud-test.ch/rest/private-openapi.json",
+    "openapi:generate:master": "generate-openapi-client.sh https://stepos-master.stepcloud-test.ch/rest/private-openapi.json",
```

Comment:
Since you already changed one of the two from unix/linux representation to windows, I think it makes sense to switch from nhy/master to linux/winodws to make it easier to use.

Reply:
I mean either this or we should revert the line back to the linux style. Mixing them with windows/linux but using the names master/nhy could be confusing

Reply:
I guess I just forgot to put it back to linux style before the commit. Fixed it